### PR TITLE
Optifine scraper

### PR DESCRIPTION
**Pull request by Ladvace**

### CODE REVIEW

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- src/common/modals/AddInstance/NewInstance.js  4
- src/common/modals/InstanceManager/Overview.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691&bid=17151374)

File: src/common/reducers/actions.js [hunk 1]
```diff
@@ -1647,6 +1810,52 @@ export function launchInstance(instanceName) {
...
+          }
+        );
+      }
+    }
+
+    const optifineVersionNameFixedFormat =
+      instanceConfig.optifine &&
+      `${instanceConfig.optifine.split(' ')[1]}-Optifine_${
+        instanceConfig.optifine.split(' ')[2]
+      }_${instanceConfig.optifine.split(' ').slice(3, 5).join('_')}`;
```
Comment (killpowa) [inline src/common/reducers/actions.js]: Codacy Issue found: [Replace `.split('·').slice(3,·5)` with `⏎········.split('·')⏎········.slice(3,·5)⏎········`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- src/common/modals/InstanceManager/Overview.js  1
- src/common/modals/AddInstance/NewInstance.js  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691&bid=17151374)

File: src/common/reducers/actions.js (hunk 1 above)
Comment (killpowa) [inline src/common/reducers/actions.js]: Codacy Issue found: [Replace `.split('·').slice(3,·5)` with `⏎········.split('·')⏎········.slice(3,·5)⏎········`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691)

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- src/common/modals/AddInstance/NewInstance.js  4
- src/common/modals/InstanceManager/Overview.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691&bid=17151374)

File: src/common/reducers/actions.js (hunk 1 above)
Comment (blarfoon) [inline src/common/reducers/actions.js:1855]: Codacy Issue found: [Replace `.split('·').slice(3,·5)` with `⏎········.split('·')⏎········.slice(3,·5)⏎········`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5231691)

**Pull request closed without merging**